sherpa-onnx/csrc/lodr-fst.h

```diff
@@ -0,0 +1,70 @@
+// sherpa-onnx/csrc/lodr-fst.h
+//
+// Contains code copied from icefall/utils/ngram_lm.py
+// Copyright (c)  2023 Xiaomi Corporation
+//
+// Copyright (c)  2025 Tilde SIA (Askars Salimbajevs)
+
+
+#ifndef SHERPA_ONNX_CSRC_LODR_FST_H_
+#define SHERPA_ONNX_CSRC_LODR_FST_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+#include <optional>
+#include <tuple>
+#include <unordered_map>
+#include <limits>
+#include <algorithm>
+#include <utility>
+
+#include "kaldifst/csrc/kaldi-fst-io.h"
+
+namespace sherpa_onnx {
+
+class Hypothesis;
+
+class LodrFst {
+ public:
+  explicit LodrFst(const std::string &fst_path);
+
+  std::pair<std::vector<int32_t>, std::vector<float>> GetNextStateCosts(
+    int state, int label);
```

Comment:
Can you use `int32_t` to replace `int`?

In sherpa-onnx, we use `int32_t` in general.